Libraries/Lists/ListView/__mocks__/ListViewMock.js

```diff
@@ -34,7 +34,7 @@ class ListViewMock extends React.Component {
         const rowID = rowIDs[rowIdx];
         rows.push(
           <StaticRenderer
-            key={rowID}
+            key={'r_' + sectionID + '_' + rowID}
```

Comment:
Please add comment here why it is required.
And I believe something more explicit like `{'section_' + sectionID + '_row_' + rowID}` would be better.